imod/tests/fixtures/mf6_circle_fixture.py

```diff
@@ -310,9 +310,9 @@ def circle_model_transport():
     transport_model["adv"] = imod.mf6.AdvectionUpstream()
     transport_model["mst"] = imod.mf6.MobileStorageTransfer(porosity)
 
-    # %%
-    # Now we define initial conditions (0.0) and output options for the transport
-    # simulation, and assign the transport model to the simulation as well.
+    # %% Define the maximum concentration as the initial conditions, also output
+    # options for the transport model, and assign the transport model to the
+    # simulation as well.
```

Comment:
See previous comment (less important because this is not rendered as html):

```suggestion
    # %% 
    # Define the maximum concentration as the initial conditions, also output
    # options for the transport model, and assign the transport model to the
    # simulation as well.
```